client/blocks/post-actions/index.jsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { partial } from 'lodash';
+import classnames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { recordGoogleEvent } from 'state/analytics/actions';
+import PostRelativeTimeStatus from 'my-sites/post-relative-time-status';
+import CommentButton from 'blocks/comment-button';
+import LikeButton from 'my-sites/post-like-button';
+import PostTotalViews from 'my-sites/posts/post-total-views';
+import utils from 'lib/posts/utils';
+
+const getContentLink = ( site, post ) => {
+	let contentLinkURL = post.URL;
+	let contentLinkTarget = '_blank';
```

Comment:
I was informed in one of my PRs where I used external links that we should add `rel="noopener noreferrer`: https://github.com/Automattic/wp-calypso/pull/7680